libs/firebase-utils/src/lib/firestoreMigrations/0074.ts

```diff
@@ -0,0 +1,53 @@
+import { Firestore } from '../types';
+import { runChunks } from '../firebase-utils';
+import { Organization, Movie } from '@blockframes/model';
+
+/**
+ * Remove isBlockchainEnabled from org documents
+ * @param db
+ * @returns
+ */
+export async function upgrade(db: Firestore) {
+  await migrateMovie(db);
+  return await migrateOrg(db);
+}
+
+async function migrateMovie(db: Firestore) {
+  const movies = await db.collection('movies').get();
+
+  return runChunks(movies.docs, async (doc) => {
+    const movie = doc.data() as Movie;
+
+    // Check if movie has a crew with mini 1 member
+    if (!movie.crew) return false;
+    if (!Object.prototype.hasOwnProperty.call(movie, 'crew')) return false;
+    if (movie.crew.length == 0) return false;
```

Comment:
I think is is enough
```suggestion
   if (!movie.crew?.length) return false;
```